packages/headless/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@sit-onyx/headless",
```

Comment:
Do we also neec to add this package to the changeset package? I've seen that there is a `initialVersion` config for all packages

Reply:
According to my tests it isn't necessary for new packages, but I added it anyways